src/pocketmine/Server.php

```diff
@@ -2686,6 +2687,9 @@ private function titleTick(){
 	public function handlePacket($address, $port, $payload){
 		try{
 			if(strlen($payload) > 2 and substr($payload, 0, 2) === "\xfe\xfd" and $this->queryHandler instanceof QueryHandler){
+				$ev = new RawPacketReceiveEvent($address, $port, $payload);
```

Comment:
Hmmm that I didn't notice.. but you should add snapse back into Genisys and/or some sort of Nemisys Impementation. Please and thanks.